apps/plea/stages.py

```diff
@@ -393,9 +393,14 @@ def __init__(self, *args, **kwargs):
         super(YourDetailsStage, self).__init__(*args, **kwargs)
 
     def save(self, form_data, next_step=None):
+        contact_number= form_data.get("contact_number")
+        contact_number = contact_number.replace(" ", "").replace("+", "00").replace(',', '').replace('.', '')\
```

Comment:
there's an easier way to do it without all the replace() calls, and more robust since you don't have to specify each character 
re.sub('\D', '', n.replace('+', '00'))